src/pages/components/text-inputs.mdx

```diff
@@ -0,0 +1,199 @@
+propSheets:
+  - forms/src/elements/Input.tsx
+  - forms/src/elements/Textarea.tsx
+  - forms/src/elements/MediaInput.tsx
+  - forms/src/elements/FauxInput.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>Use an Input to let the user enter text</li>
+      <li>Use a Textarea for multiline text</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>For read-only objects, use text</li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## Examples
+
+### Input
+
+A text input is a form field that lets the user enter text.
+
+import Input from '../../examples/text-inputs/Input.tsx';
+import InputCode from '!!raw-loader!../../examples/text-inputs/Input.tsx';
+
+<CodeExample code={InputCode}>
+  <Input />
+</CodeExample>
+
+### Textarea
+
+A Textarea accommodates multiline text. Its height is fixed by default, and can be resizable using
+the `isResizable` property.
+
+import Textarea from '../../examples/text-inputs/Textarea.tsx';
+import TextareaCode from '!!raw-loader!../../examples/text-inputs/Textarea.tsx';
+
+<CodeExample code={TextareaCode}>
+  <Textarea />
+</CodeExample>
+
+### MediaInput
+
+A MediaInput can have embedded icons, to further clarify its purpose.
+
+import Icons from '../../examples/text-inputs/MediaInput.tsx';
+import IconsCode from '!!raw-loader!../../examples/text-inputs/MediaInput.tsx';
+
+<CodeExample code={IconsCode}>
+  <Icons />
+</CodeExample>
+
+### FauxInput
+
+A FauxInput makes non-inputs look like inputs.
+
+import FauxInput from '../../examples/text-inputs/FauxInput.tsx';
+import FauxInputCode from '!!raw-loader!../../examples/text-inputs/FauxInput.tsx';
+
+<CodeExample code={FauxInputCode}>
+  <FauxInput />
+</CodeExample>
+
+### Regular weight label
+
+Text input labels are emphasized by default. They can be regular weight using the `isRegular`
+property.
+
+Regular labels are not recommended for text inputs, as they flatten visual hierarchy making
+scanning more difficult.
+
+import RegularWeightLabel from '../../examples/text-inputs/RegularWeightLabel.tsx';
+import RegularWeightLabelCode from '!!raw-loader!../../examples/text-inputs/RegularWeightLabel.tsx';
+
+<CodeExample code={RegularWeightLabelCode}>
+  <RegularWeightLabel />
+</CodeExample>
+
+### Hint
+
+Hint adds additional context to the text Input’s label.
+
+import Hint from '../../examples/text-inputs/Hint.tsx';
+import HintCode from '!!raw-loader!../../examples/text-inputs/Hint.tsx';
+
+<CodeExample code={HintCode}>
+  <Hint />
+</CodeExample>
+
+### Size
+
+A text Input can be default or compact.
+
+import Size from '../../examples/text-inputs/Size.tsx';
+import SizeCode from '!!raw-loader!../../examples/text-inputs/Size.tsx';
+
+<CodeExample code={SizeCode}>
+  <Size />
+</CodeExample>
+
+### Bare
+
+Input borders and padding can be removed using the `isBare` property. Use an
+[aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)
+to label the input.
```

Comment:
> Use an aria-label

Why? Also not reinforced by the example.

Reply:
> Why? Also not reinforced by the example.

I brought this up in the Google doc and I am hoping to get the correct copy soon. Then the PR can align with the Google doc. I think this sentence about `aria-label` will be deleted.

Reply:
We talked about this in the doc review; I was showing a `bare` input with a label, and the team let me know that `bare` inputs typically have a hidden label and an aria-label. I left a comment to fix the image in the final asset. This image was only a placeholder.

Either way sounds good to me though!

Reply:
Oh, this is an example, right, there would be no final asset. Should have just corrected in the doc—I’ll do that next time. Sorry for the confusion.

Reply:
If the majority case is that a bare input has no label, then maybe we should show that? But let's do it the right way. There is no need for `aria-label`. The technique would be to use `<Label hidden={true}>` – allowing the VO to read out the associated label when the bare input is focused.

Reply:
Awesome sounds good, removed the `aria-label` copy.